explore/src/main/scala/explore/config/SpectroscopyModesTable.scala

```diff
@@ -97,7 +97,7 @@ final case class SpectroscopyModesTable(
 }
 
 object SpectroscopyModesTable {
-  type Props = SpectroscopyModesTable
+  private type Props = SpectroscopyModesTable
 
   type ColId = NonEmptyString
```

Comment:
All members can probably be `private` too.

Reply:
WDYT about declaring the whole `SpectroscopyModesTable` private. It will achieve the same and apply to everything in one go
See my last commit

Reply:
I think it's a good idea to declare the object private, but we should also make members private. Otherwise, they are still accessible from within the same package, which in most cases we don't want to allow.